catalog/tests/test_models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from catalog.models import Dish, DishType
+
+
+class ModelTests(TestCase):
```

Comment:
1. Test czy defaultowa wartość dla `years_of_experience` ustawi się na 0. Gdy ktoś spróbuje to zmienić, zmiana ta będzie widoczna od razu w testach.
2. Również test na ujemną cenę. 
3. Test na brakujące required pola